sql/core/src/main/scala/org/apache/spark/sql/RuntimeConfig.scala

```diff
@@ -64,7 +64,7 @@ class RuntimeConfig private[sql](sqlConf: SQLConf = new SQLConf) {
   /**
    * Sets the given Spark runtime configuration property.
    */
-  protected[sql] def set[T](entry: ConfigEntry[T], value: T): Unit = {
+  private[sql] def set[T](entry: ConfigEntry[T], value: T): Unit = {
```

Comment:
Other places have been `protected[sql]` since 2016. Do we also need to weaken the access scope?

Reply:
I can't find any difference between `private[sql]` and `protected[sql]` since users can't extend `RuntimeConfig`.

Reply:
Got it